Makefile.am

```diff
@@ -189,7 +189,7 @@ $(BITCOIN_WALLET_BIN): FORCE
 	$(MAKE) -C src $(@F)
 
 if USE_LCOV
-LCOV_FILTER_PATTERN=-p "/usr/include/" -p "/usr/lib/" -p "src/leveldb/" -p "src/bench/" -p "src/univalue" -p "src/crypto/ctaes" -p "src/secp256k1"
+LCOV_FILTER_PATTERN=-p "/usr/include/" -p "/usr/lib/" -p "src/leveldb/" -p "src/bench/" -p "src/univalue" -p "src/crypto/ctaes" -p "src/secp256k1" -p "depends"
```

Comment:
style-nit: Could make this multiline to avoid very long lines?

Reply:
Will fix! :+1:

Reply:
Right, usually the one-per-line approach is preferred for these, not only because long lines are inconvenient but also as it makes it easier to review as well as rebase changes